phoenix-core/src/main/java/org/apache/phoenix/iterate/ExplainTable.java

```diff
@@ -120,7 +120,7 @@ protected void explain(String prefix, List<String> planSteps) {
         } else {
             explainSkipScan(buf);
         }
-        buf.append("OVER ").append(tableRef.getTable().getPhysicalName().getString());
+        buf.append("OVER ").append(tableRef.getTable().getPhoenixPhysicalName().getString());
```

Comment:
getPhoenixPhysicalName() is required to get phoenix table name (X.Y) , this is different from getName() as in case of view, it return phoenix name of the data table ,which sometimes required for "Explain" plan or sometimes to get tableRef for data table. I think , we can have different name for this method to avoid confusion.

getPhysicalName gives the hbase representation(X:Y) of table/view depending upon isNamespaceMapped flag.